docs/deployment/helm/index.md

```diff
@@ -19,6 +21,26 @@ All the available charts can be found in [artifacthub.io](https://artifacthub.io
 
 While Ubuntu 20.04/22.04 was used across all three CSP environments, it's worth noting that node images are not limited to Ubuntu 20.04/22.04 only. The majority of widely recognized Linux distributions should be suitable for use.
 
+## Unstable Helm Charts
+
+Helm charts are also published from the main/development branch after each merge.
+These charts reference the latest development images tagged as `unstable` and are
+are stored alongside plugin images in the OCI image registry.
+
+### Discovering Unstable Helm Charts
+
+Unstable charts can be discovered using [skopeo](https://github.com/containers/skopeo).
+For instance, one can list the available charts for the balloons plugin using this
+skopeo command:
+`skopeo list-tags docker://ghcr.io/containers/nri-plugins/nri-resource-policy-balloons`
+
+### Using Unstable Helm Charts
+
+Once discovered, unstable Helm charts can be used like any other. For instance, to use
+the `0.3.2-unstable` version of the chart to install the development version of the
```

Comment:
We don't have unstable for the release branches so maybe we should just refer to the main branch here? I.e.`v0.4.0-unstable`

Reply:
Yes, but then I guess it would be better to not spell out any particular version. So maybe I'll update it as `$X.$Y-unstable` and trust the user to figure out what that means in practice.